lms/djangoapps/sneakpeek_deeplink/middleware.py

```diff
@@ -19,6 +22,8 @@
     'info',
 )
 
+log = logging.getLogger("edx.sneakpeek_middleware")
```

Comment:
And it looks like quality checks were disabled from the test suite.
I have restored this check, so subsequent builds will be checked
for quality violations (pep8, pylint).